iiwa_tools/CMakeLists.txt

```diff
@@ -22,7 +22,7 @@
 #|    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 #|    GNU General Public License for more details.
 #|
-cmake_minimum_required(VERSION 3.1.0)
+cmake_minimum_required(VERSION 3.5)
```

Comment:
Why 3.1.0 is not enough?

Reply:
This was not intended, must have been a holdover from something else I was trying. I'll remove it.